app/main.py

```diff
@@ -16,4 +16,12 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable" : [my_favourite_films,
+                 marks,
+                 collection_of_coins],
+    "immutable": [lucky_number,
+                  pi,
+                  one_is_a_prime_number,
+                  name,
+                  profile_info]}
```

Comment:
Fix indentation here, according to the best practices
Braces should be located in one of two options: open and start with the text or have a line break between the text